hypha/static_src/src/app/src/containers/DisplayPanel/index.js

```diff
@@ -69,10 +68,7 @@ const DisplayPanel = props => {
         </Tab>,
         <Tab button="Notes" key="note">
             <NoteListing submissionID={submissionID} />
-            {isEditing ? (
-                    <EditNoteForm submissionID={submissionID}/>
-
-                ) : (
+            {isEditing ? null : (
```

Comment:
this makes sure that the comment submit box disappears when a user is editing a comment